test/kube_testing/k8s.go

```diff
@@ -301,6 +303,13 @@ func (o *K8s) deletePodForce(pod *v1.Pod) error {
 	return nil
 }
 
+func (o *K8s) initNamespace() {
+	var err error
+	nsmNamespace := os.Getenv("NSM_NAMESPACE")
```

Comment:
Constant somewhere in k8s/pkg, please.

Reply:
So what if the env is empty or does not exist?

Reply:
In this case `nsmNamespace` will be empty and `default` namespace will be used